plugin/src/main/kotlin/tanvd/grazi/ide/quickfix/GraziReplaceTypo.kt

```diff
@@ -42,7 +46,17 @@ class GraziReplaceTypo(private val typo: Typo) : LocalQuickFix, Iconable, Priori
             editor.selectionModel.setSelection(selectionRange.startOffset, min(selectionRange.endOffset, editor.document.textLength))
 
             val items = typo.fixes.map { LookupElementBuilder.create(it) }
-            LookupManager.getInstance(project).showLookup(editor, *items.toTypedArray())
+            LookupManager.getInstance(project).showLookup(editor, *items.toTypedArray())?.apply {
```

Comment:
Let's abstract it to private local `object` with name something like `object LookupFUCounterCollector`

Reply:
It can't be object because it use local variable typo. I can make extension function, if you think it would be better.

Reply:
Hm, true. Let's make it an extension function :)

Reply:
Done